boards/arm/mimxrt1050_evk/Kconfig.defconfig

```diff
@@ -94,4 +94,17 @@ endchoice
 
 endif # LVGL
 
+if DISPLAY_MCUX_ELCDIF
```

Comment:
This should be `if CHARACTER_FRAMEBUFFER`

Reply:
updated, Thanks